organization/_posts/2024/2024-02-29-coordination.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "HSF Coordination Meeting #265, 29 February 2024"
+layout: plain_toc
+---
+
+## Attending
+
+Present/Contributing: Caterina Doglioni, Pere Mato, Jim Pivarski, Reiner Hauser, Joe Osborn, Alexander Moreno, Andrea Valenzuela, Liz Sexton-Kennedy, Paul Laycock, Patrick Gartung, Jamie Gooding, Benedikt Hegner, Matthew Feickert, Torre Wenaus, Michel Jouvin, Stefan Roiser, Tommaso Lari, Krzysztof Genser, Steven Mrenna, Daniel Elvira
+
+Apologies/Contributing: Graeme Stewart, Eduardo Rodrigues
+
+## News, general matters, announcements
+
+### HSF Joint Workshop with WLCG
+
+Registration is now open for the WCLG-HSF workshop at DESY (13-17 May) is now available: <https://indico.cern.ch/e/wlcg-hsf-2024> (register at the DESY linked Indico)
+
+Graeme, Eduardo and Caterina are helping with the overall organisation for HSF.
+
+We would like to have a few dedicated sessions/talks:
+
+- Analysis Facilities (N.B. the whitepaper is now finalised), discussing with the experiments
+- Software Citation and Recognition Follow-up
+    - Caterina and Stefan follow up with EVERSE recognition at the Kick-off meeting
+- HSF Evolution Plans
+- HSF Training Activities
+
+Our main HSF activity will, however, refocus on community software. We have opened a [call for abstracts](https://indico.cern.ch/event/1369601/abstracts/):
+
+> The HSF will have several sessions on Community Software at the workshop.
+We invite you to submit abstracts for talks of 20' where you can describe a tool, package or other contribution that you are making to software for multiple HEP experiments.
+Submissions describing projects or other ideas, such as training initiatives or community support structures are also warmly welcomed.
+This is your chance to show off how good your project is and to meet and talk to current and future users!
+
+**Please broadcast this call widely and encourage contributors.** Deadline is 18 March.
+
+### LHCC
+
+Generators presented at the last LHCC meeting, with a number of comments/findings from the HSF discussions. Echoed from the generator workshop from last November: 
+- need better pipelines for generator software development
+- [see also workshop conclusions]
```

Comment:
Is there some link that should go here?

Reply:
Hmm, I can't find the "November workshop" mentioned there on indico or on the generators WG page - it's been a long week so I may just be unable to search, but if @sextonkennedy has an idea of which workshop was mentioned here I'll add the link.